README.md

```diff
@@ -1,2 +1,95 @@
+[![Codecov](https://img.shields.io/codecov/c/github/semantic-release/github.svg)](https://codecov.io/gh/semantic-release/github)
+[![Greenkeeper badge](https://badges.greenkeeper.io/semantic-release/github.svg)](https://greenkeeper.io/)
+
+## verifyConditions
+
+Verify the presence and the validity of the `githubToken` (set via option or environment variable).
+
+### Options
+
+| Option                | Description                                 | Default                                                |
+| --------------------- | ------------------------------------------- | ------------------------------------------------------ |
+| `githubToken`         | The token used to authenticate with GitHub. | `process.env.GH_TOKEN` or `process.env.GITHUB_TOKEN`   |
+| `githubUrl`           | The GitHub Enterprise endpoint. Optional.   | `process.env.GH_URL` or `process.env.GITHUB_URL`       |
+| `githubApiPathPrefix` | The GitHub Enterprise API prefix. Optional. | `process.env.GH_PREFIX` or `process.env.GITHUB_PREFIX` |
+
+## publish
+
+Publish a [Github release](https://help.github.com/articles/about-releases).
+
+### Options
+
+| Option                | Description                                           | Default                                                |
+| --------------------- | ----------------------------------------------------- | ------------------------------------------------------ |
+| `githubToken`         | The token used to authenticate with GitHub.           | `process.env.GH_TOKEN` or `process.env.GITHUB_TOKEN`   |
+| `githubUrl`           | The GitHub Enterprise endpoint. Optional.             | `process.env.GH_URL` or `process.env.GITHUB_URL`       |
+| `githubApiPathPrefix` | The GitHub Enterprise API prefix. Optional.           | `process.env.GH_PREFIX` or `process.env.GITHUB_PREFIX` |
+| `assets`              | An array of files to upload to the release. Optional. | - 
+
+#### assets option
+
+Each element of the array can be a path to the file or an `object` with the properties:
+
+| Property | Description                                                              | Default                              |
+| -------- | ------------------------------------------------------------------------ | ------------------------------------ |
+| `path`   | The file path to upload relative to the project directory. Required      | -                                    |
+| `name`   | The name of the downloadable file on the Github release. Optional.       | File name extracted from the `path`. |
+| `label`  | Short description of the file displayed on the Github release. Optional. | -                                    |
+
+## Configuration
+
+The plugins are used by default by [semantic-release](https://github.com/semantic-release/semantic-release) so no specific configuration is requiered if `githubToken`, `githubUrl` and `githubApiPathPrefix` are set via environment variable and if `assets` is not required.
```

Comment:
I would leave out the "and if `assets` is not required." part, I think it’s clear from the `Options` table above

Reply:
Done!